Project21/Project21/Assignment 3.c

```diff
@@ -0,0 +1,46 @@
+////// ------------------------------------------------------------------------------------------------
+//// Assignment 3 ~ A training program for new joiners at Metamation, Batch - July 2024.
+//// Copyright (c) Metamation India.
+//// ------------------------------------------------------------------
+//// Assignment 3
+//// Decimal to Binary conversion and Decimal to HexaDecimal conversion
+//// ------------------------------------------------------------------------------------------------
+#include<stdio.h>
+#define INT_SIZE sizeof(int) * 8

```

Comment:
Instead of fixing the size to 32-bits, I would suggest to check the size needed to represent the given number. Say 8 can be represented in 4-bits. So this takes up more memory.